### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by TheDutchMC04** [mamorgen/magbot3#8]
**Describe the bug**
I am unable to login with Magister, due to an invalid password error, even though it is 100% correct.
Issue was posted on 7Kasper's page, but this is where the code is so I think it's more relevant here.

**To Reproduce**
1. Go to register
2. as school use a school that uses Sivog
3. fill out all the fields

**Expected behavior**
Being able to login
**Screenshots**
-
**Desktop (please complete the following information):**
 - OS: tried on ChromeOS, Windows, Ubuntu
 - Browser Chrome

**Smartphone (please complete the following information):**
 - Version 78.0.3889.0 (Official Build) dev

**Additional context**
I think the problem is related to Sivog, as when I go to sv.magister.net in my browser, I enter my username, consisting of student ID + @vituscollege.nl, then the next page I am redirected to Sivog, instead of the usual password page

**Comment by Sj3rd** [comment]
Ik heb even snel gekeken, en heb dit nog nooit eerder gezien (dat de magisterinlog verloopt via een openid systeem van de school). Ik ben bang dat dit niet makkelijk op te lossen is. Ik kan kijken of ik binnenkort tijd heb om een aangepaste versie van magbot te maken die dit probleem voor jouw school oplost, maar heb het ook erg druk met andere dingen. 

Één ding moet ik dan even weten, nadat de logingegevens op de sivog pagina zijn ingevuld, wordt je dan direct geredirect naar magister? Of ga je nog via een ander(e) pagina/systeem?

**Comment by TheDutchMC04** [comment]
Hardstikke bedankt voor je snelle antwoord als eerst.

Je wordt direct doorgeleid ja. Ongeveer zo: Magister > Sivog > Magister.
Bij de eerste magister alleen de username, zoals standaard, bij Sivog
username en password.

Ook geen haast hoor :)

Op wo 4 sep. 2019 00:18 schreef Sj3rd <notifications@github.com>:

**Comment by TheDutchMC04** [comment]
Voor de redirect: na Sivog kom je op een magister /redirect iets, maar wel in het sv.magister.net domein, hierna ga je door naar de home pagina

**Comment by Jamie4224** [comment]
Ik heb dit probleem ook. Sinds ongeveer een week is het inloggen op magister bij onze school veranderd. De school had altijd al een eigen loginportaal, voor Office, schoolmail en roostersysteem. Nu gebeurt de login voor magister ook door dit portaal.

Ik ga naar magister, voor school in en stamnr. in, zodra ik op enter druk bij het stamnr. word ik doorgestuurd naar het portaal van mijn school, de gebruikersnaam staat dan al voor mij klaar en het wachtwoord moet ik dan nog invoeren. Zodra ik dit doe en doorklik ga ik weer terug naar magister. 

Dit heeft er blijkbaar voor gezorgd dat mijn magister niet meer gesynchroniseerd wordt met calendar. Ik wist dat het inloggen niet meer kon. (Magistat werkte het inloggen niet meer) Had eigenlijk verwacht dat mn magbot zou blijven werken, omdat dit vaak via API Keys werkt.

**Comment by 7kasper** [comment]
@Jamie4224 het kan zijn dat magbot er helemaal uitligt hierdoor. Wij zijn momenteel in overleg met magister over of en hoe we verder kunnen. Wellicht kan ik het probleem nog verhelpen om de requests door te sturen zoals de magsiter app dat zou doen. Ik neem aan dat je daar wel in kan loggen?

**Comment by TheDutchMC04** [comment]
Hi,

Inloggen op de magister app gaat gewoon ja, maar ook via Sivog in mijn
geval. Apps zoals Studeo werken ook gewoon.

Op ma 9 dec. 2019 21:33 schreef 7kasper <notifications@github.com>:

**Comment by Jamie4224** [comment]
> @Jamie4224 het kan zijn dat magbot er helemaal uitligt hierdoor. Wij zijn momenteel in overleg met magister over of en hoe we verder kunnen. Wellicht kan ik het probleem nog verhelpen om de requests door te sturen zoals de magsiter app dat zou doen. Ik neem aan dat je daar wel in kan loggen?

Ja, magister app wordt je net zoals in de browser doorgestuurd en daarna weer teruggestuurd naar magister. Top dat jullie eraan werken!
